src/sentry/static/sentry/app/components/events/contexts/state.tsx

```diff
@@ -20,23 +21,40 @@ type Props = {
 };
 
 class StateContextType extends React.Component<Props> {
+  getStateTitle(name: string, type?: string) {
+    return name + (type ? ` (${upperFirst(type)})` : '');
```

Comment:
```suggestion
    return `${name} ${type ? `(${upperFirst(type)})` : ''}`;
```

Reply:
I slightly changed your suggestion so that there's no empty space after name if no type is passed. 👍
```suggestion
    return `${name}${type ? ` (${upperFirst(type)})` : ''}`;
```